rest/route/subscription_routes.go

```diff
@@ -68,7 +69,7 @@ func (s *subscriptionPostHandler) ParseAndValidate(ctx context.Context, r *http.
 			}
 		}
 
-		if dbSubscription.Owner != u.Username() {
```

Comment:
Can we also assert that OwnerType is OwnerTypePerson (since the project admin route is for changing those).